Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,870 @@
+Doch auch die Hominini besitzen diese Fähigkeit, allerdings ist ihnen Elektrizität oder im weiteren Sinne noch kein Begriff, weshalb das Ausbauen und Ausnutzen dieser Kontrolle recht schlecht ist.
+
+\subsubsection{Bedeutung im Spiel}
+Die \npref{sec:mc-diplomatin} hat diese Eigenschaft von Beginn an... oder?
+
+\subsubsection{Intuitive Nutzung}
+Die meisten Lebewesen nutzen kleine Schockstöße zur Abwehr von Räubern und Fraßfeinden. 
+Es gibt einige Pflanzen in besonders kargen Gegenden, die gelernt haben, dies als eine Art Metalldetektor einzusetzen, um wichtige Nährstoffe im Boden zu finden.
+
+\subsubsection{Erlernbare Steigerungen}
+\begin{outline}
+	\1 Metallische (magnetische) Gegenstände von entsprechender Größe sind frei bewegbar.
+	\1 Metalle werden (ent-)magnetisiert.
+	\1 Durch Berührung längeres Schocken und auch Verbrennen von Personen (nicht den ganzen Körper, sondern Berührungsstellen).
+	\1 ...
+\end{outline}
+
+\subsubsection{Meisterliche Beherrschung} 
+\begin{outline}
+	\1 Ähnlich wie beim Gleichstrom kann der Wechselstrom als Defibrillator eingesetzt werden - hier allerdings nur, um die Herzmuskeln zur Kontraktion zu bringen.
+	Kann sich das System dann nicht weiter selbst erhalten, ist nichts mehr zu machen.
+	(... Frankensteins Monster... ?) %todo
+	\1 Anders herum kann ohne Berührung Strom im anderen Körper erzeugt werden.
+	\1 ...
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+\section{Die Genetik der Magie}
+Wie zuvor angesprochen ist das Nutzen der Magie evolutionär entstanden und dementsprechend vererbbar.
+Der Einfachheit halber wird mit der mendelschen Vererbung über Allele gearbeitet.
+D.h. in jedem Wesen gibt es \textbf{zwei Allele}: je ein Allel von der Mutter und eines vom Vater.
+Diese nennt man den \textbf{Genotypen}: er stellt dar, was an genetischem Material vorhanden ist.
+Abhängig von der "`Durchsetzungskraft"' der beiden Allele kommt es nun zu einer bestimmten \textbf{Ausprägung} der elterlichen Gaben beim Nachkommen - also auf welche Art er Magie nutzen kann.
+Dies nennt man den \textbf{Phänotypen}: er zeigt auf, was am Ende ausgebildet wird.
+
+Bei Bakterien ist das alles komplett anders, aber hier nicht von Belang.
+
+
+\subsection{Allele der Menschen unseres Landes} %todo hübsche Tabelle + Beschriftung!
+Die X %todo
+unter den Menschen in \nameref{sec:land} verbreiteten Arten der Kontrolle über Magie finden sich in Tab. \ref{tab:landmagie}.
+Von den dargestellten Magiearten nutzt der Soldat Temperatur, %todo
+die Spionin Licht %todo
+und die Diplomatin zuerst Druck und später auch Elektrizität. %todo
+
+\begin{table}[hbtp]
+	\centering
+	\caption[Verbreitete Magiearten in \nameref{sec:land}]%
+	{\textbf{Die Arten der Magiekontrolle, die unter den Menschen aus \nameref{sec:land} verbreitet sind. } \\
+		Die Tabelle liest sich zeilenweise wie folgt: gibt es ein eigenes Allel für die Magieart, so findet sich ein Eintrag in reinerbig.
```

Comment:
reinerbig iwie in Anführungszeichen oder so, damit klar ist, dass das ein Begriff in der Tabelle gemeint ist
genauso bei weiteren Erläuterungen

Reply:
aye